src/lib/block_if.c

```diff
@@ -372,7 +389,7 @@ blockif_proc(struct blockif_ctxt *bc, struct blockif_elem *be, uint8_t *buf)
```

Comment:
I think I like that, ie either `__FreeBSD__` or feature (if appropriate). Will update again...though it looks like GH was royally confused with my force push which also updated the author and committer. Might have to open a new PR